Add new channel flag (OBFUSCATED) (#287)
discord-userdoccers/discord-userdoccers: pages/resources/channel.mdx

| 1 << 14     | BROADCASTING                               | This group DM is used for broadcasting a live stream                                                                                                            |
| 1 << 15     | HIDE_MEDIA_DOWNLOAD_OPTIONS                | This media channel has the embedded download options hidden for media attachments                                                                               |
| 1 << 16     | JOIN_REQUEST_INTERVIEW_CHANNEL             | This group DM is used for [guild join request interviews](https://support.discord.com/hc/en-us/articles/23187611406999-Guilds-FAQ#h_01HXW2MCD0BT70FB9SRV1YRPBV) |

> | 1 << 17     | OBFUSCATED                                 | This channel's name is obfuscated because user doesn't have access to it                                                                                        |

Reviewer: ```suggestion
| 1 << 17     | OBFUSCATED                                 | This channel is unavailable for the current user                                                                                        |
```

Can you add a footnote saying something like ``The channel's name is always returned as `___hidden___`.``

| 1 << 14     | BROADCASTING                               | This group DM is used for broadcasting a live stream                                                                                                            |
| 1 << 15     | HIDE_MEDIA_DOWNLOAD_OPTIONS                | This media channel has the embedded download options hidden for media attachments                                                                               |
| 1 << 16     | JOIN_REQUEST_INTERVIEW_CHANNEL             | This group DM is used for [guild join request interviews](https://support.discord.com/hc/en-us/articles/23187611406999-Guilds-FAQ#h_01HXW2MCD0BT70FB9SRV1YRPBV) |

> | 1 << 17     | OBFUSCATED ^2^                             | This channel is unavailable for the current user                                                                                                                |

Reviewer: ```suggestion
| 1 << 17     | OBFUSCATED ^2^                             | This channel is unavailable for the current user, and has its details obfuscated                                                                                                                |
```

^1^ Media channels are now represented by the [`GUILD_MEDIA` channel type](#channel-type).

> ^2^ The channel's name is always returned as `___hidden___`.

Reviewer: ```suggestion
^2^ The channel's name and topic are always returned as `___hidden___`.
```
